### PR TITLE
#9Task.alex.moiseyenko 

### PR DESCRIPTION
**Pull request by alexmoiseyenko**
https://travis-ci.org/github/alexmoiseyenko/js-assignments/builds/714197740

### CODE REVIEW

**Comment by KostyaKov** [comment]
The tests do not contain two points:
1) logger method should log start and end of call of the standard js function
2) logger method should log start and end of call of the specified function

Can you add them, please?

File: task/09-functions-n-closures-tasks.js [hunk 1]
```diff
@@ -171,7 +193,8 @@ function partialUsingArguments(fn) {
  *   getId10() => 11
  */
 function getIdGeneratorFunction(startFrom) {
-    throw new Error('Not implemented');
+    let id = startFrom;
```
Comment (KostyaKov) [inline task/09-functions-n-closures-tasks.js]: You can solve this task without variable

**Review by KostyaKov — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by alexmoiseyenko** [comment]
https://travis-ci.org/github/alexmoiseyenko/js-assignments/builds/722862270

**Review by KostyaKov — COMMENTED** [review]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**